src/components/addSourceWizard/hardcodedComponents/azure/costManagement.js

```diff
@@ -202,10 +208,77 @@ export const ExportSchedule = () => {
         <TextListItem component={TextListItemVariants.dd}>
           {intl.formatMessage({
             id: 'cost.azure.createdAccountName',
-            defaultMessage: 'Storage account name created earlier',
+            defaultMessage: 'Created storage account name or existing storage account name',
+          })}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          <Text component={TextVariants.b}>
+            {intl.formatMessage({
+              id: 'cost.azure.resourceGroupName',
+              defaultMessage: 'Resource group name',
+            })}
+          </Text>
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {intl.formatMessage({
+            id: 'cost.azure.createdResourceGroupName',
+            defaultMessage: 'Resource group for the storage account',
           })}
         </TextListItem>
       </TextList>
     </TextContent>
   );
 };
+
+export const ExportScope = () => {
+  const intl = useIntl();
+
+  return (
+    <TextContent>
+      <Text component={TextVariants.p}>
+        {intl.formatMessage({
+          id: 'cost.azure.costExportScopeDescrption',
+          defaultMessage: 'From the Azure portal, select the scope for the new cost export.',
+        })}
+      </Text>
+      <TextContent className="list-align-left">
+        <TextList component={TextListVariants.ul}>
+          <TextListItem component={TextListItemVariants.li}>
+            {intl.formatMessage({
+              id: 'cost.azure.scopeDescription',
+              defaultMessage:
+                'For a cost export for a billing profile, provide the "id" for the billing profile from the command below.',
```

Comment:
this step is confusing. Does it mean that I need to provide an id of a billing profile for a cost export for that specific billing profile?